### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by fsfrazao** [fsfrazao/DegDay#3]
Hey everyone!
I'll just leave this link to google's style guide here:
https://google.github.io/styleguide/pyguide.html#Comments

Have a look at the **comments** part.
It would be great if we could write our code (and docstrings) following the same style and I think theirs is a good one.

Later we could make sure our code passes flake8 or other style checker.

**Comment by djo504** [comment]
@fsfrazao @cj4755 does the canada map need to run from commandline too like the others? Thanks

**Comment by djo504** [comment]
@cj4755 I will be done latest by 2pm today. I am sorry for keeping this long. Just a tiny bit of work is left to be done please bear with me

**Comment by fsfrazao** [comment]
@djo504, yes it does, since @cj4755 needs to make the Makefile run all of the tools we are writing

**Comment by djo504** [comment]
@cj4755  I am commiting right away

**Comment by djo504** [comment]
@cj4755 @fsfrazao  the map code has been commited. I am trying to plot the legend at the moment. While at that @cj4755 can continue with the makefile integration. I hope i will be able to plot the legend on time
